lib/Command/SetupCommand.php

```diff
@@ -91,7 +91,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$requirements = new Requirements($this->modelService, $this->modelVersion);
 		if ($requirements->modelFilesPresent()) {
 			$this->logger->writeln('The models are already installed');
-			return 0;
```

Comment:
HACK: modelFilesPresent() now is hacked to always print fullpath, so it is always true.. So, force reinstall here..